cmd/crane/cmd/serve.go

```diff
@@ -54,20 +54,27 @@ Contents are only stored in memory, and when the process exits, pushed data is l
 			if port == "" {
 				port = "0"
 			}
-			listener, err := net.Listen("tcp", ":"+port)
+			listenOn := ":" + port
+			if address != "" {
+				listenOn = address
+			}
+
+			listener, err := net.Listen("tcp", listenOn)
 			if err != nil {
 				log.Fatalln(err)
 			}
 			porti := listener.Addr().(*net.TCPAddr).Port
 			port = fmt.Sprintf("%d", porti)
 
 			bh := registry.NewInMemoryBlobHandler()
-			if disk {
-				tmp := os.TempDir()
-				log.Printf("storing blobs in %s", tmp)
-				bh = registry.NewDiskBlobHandler(tmp)
+			if cmd.Flags().Changed("blobs-to-disk") {
+				path := os.TempDir()
```

Comment:
Should this be [`MkdirTemp`](https://pkg.go.dev/os#MkdirTemp) ?

Reply:
hmm maybe, maybe I didn't change this at all but happy to change now.

Reply:
yeah, that's better. i changed it to mkdirtemp :)